### DISCUSSION TITLE
I set 'metricsGenerator' to 'true' but want to send metrics to grafana-mimir. can we do that. Please help?

### DISCUSSION THREAD

**Discussion by Shindek77** [grafana/tempo#2509] [Q&A]
I want send the metrics from temp to grafana-mimir using remotewrite instead of prometheus.
Just want to know can we do that. If yes what will be metrics I can check

**Comment by mapno** [comment]
Hi @Shindek77. The metrics-generator can export metrics to any Prometheus-compatible backend, which includes Grafana Mimir. All you need to change is the endpoint and credentials to Mimir if you're already sending metrics to Prometheus.

For remote-write, Tempo exports all the metrics from the Prometheus Agent that is embedded. You can check `prometheus_remote_storage_bytes_total` to see if sampled are being sent.

**Reply by Shindek77** [reply]
@mapno thank you for your reply. Actually there is no prometheus in our stack we have tempo, mimir and grafana only.  I am giving configuration on tempo as follows:
# Configuration for the metrics-generator
metricsGenerator:
  # -- Specifies whether a metrics-generator should be deployed
  enabled: true
  config:
    storage:
      path: /var/tempo/wal
      wal:
      remote_write_flush_deadline: 1m
      # -- A list of remote write endpoints.
      remote_write:
        - url: http://mimir-test.*********/api/v1/push
Also you have mentioned about credentials. Which credentials could you explain please? Also where should I check prometheus_remote_storage_bytes_total to see if sampled are being sent..
